### PR TITLE
Pass --add to asdf install to add plugins in .tool-versions

### PR DESCRIPTION
**Pull request by trs**
# Summary

Normally, `asdf install` fails when encountering a plugin in `.tool-versions` that does not exist.

This allows the flag `--add` to be passed which adds the plugins before installing.

## Usage:

`asdf install --add`

### CODE REVIEW

**Comment by trs** [comment]
@forstermatth What do you think?

File: lib/commands/install.sh [hunk 1]
```diff
@@ -59,6 +73,17 @@ install_local_tool_versions() {
   fi
 }
 
+install_add_tool_plugin() {
+  local plugin_name=$1
+  local plugin_path
+  plugin_path=$(get_plugin_path "$plugin_name")
+
+  mkdir -p "$(asdf_dir)/plugins"
```
Comment (forstermatth) [inline lib/commands/install.sh]: Is this necessary if it's already performed in the `plugin_add_command`?

Comment (trs) [reply lib/commands/install.sh]: The reason I added this was so the program doesn't exit.

`plugin_add_command ` exits if the plugin was already added: https://github.com/trs/asdf/blob/master/lib/commands/plugin-add.sh#L29

Comment (trs) [reply lib/commands/install.sh]: Sorry, I didn't realize this at first. You're right, the `mkdir` is not needed.

**Pull request merged**